parser/src/json/compiler.rs

```diff
@@ -614,52 +614,57 @@ impl Compiler {
             items.push((seq, false));
         }
         let opener = self.builder.string("{");
-        let inner = self.ordered_sequence(&items, false);
+        let inner = self.ordered_sequence(&items, false, &mut HashMap::<(&[(NodeRef, bool)], bool), NodeRef>::new());
```

Comment:
&mut HashMap::new() should work without types?

Reply:
Was having trouble with an earlier state of this code -- hopefully it can do the type inference now 😆